soc/espressif/esp32c3/mcuboot.ld

```diff
@@ -133,6 +133,7 @@ SECTIONS
     . += 16;
 
     _text_end = ABSOLUTE(.);
+    __text_region_end = .;
```

Comment:
@marekmatej after #70505 (7ecc715722c54881e7ae82546f5cc212afcc3fda), there's `_text_end` in the `zephyr/soc/espressif/esp32c3/mcuboot.ld` but the `_text_start` is now gone, could you please advise how to do this now? Cheers.

Reply:
Hi @ycsin, is the symbol needed to be in the `mcuboot.ld` at all?
I made a mistake by removing it from the `default.ld`. Please take a look at the fix https://github.com/zephyrproject-rtos/zephyr/pull/71388

Reply:
Thanks for that PR.

> Hi @ycsin, is the symbol needed to be in the `mcuboot.ld` at all?

Looks like it is used in MCUBOOT image only, probably not required.